src/interface/swagger-handler.json

```diff
@@ -0,0 +1,37 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2.0",
+        "title": "MOJALOOP Central Settlements",

```

Comment:
Probably should change the title and description?

This is more an Ops API for handlers right?